src/main/java/seedu/address/ui/PomodoroDisplay.java

```diff
@@ -17,10 +19,10 @@
     @FXML private Label taskInProgress;
     @FXML private Label timer;
 
-    public PomodoroDisplay(String taskInProgressText, String timerText) {
+    public PomodoroDisplay() {
         super(FXML);
-        this.taskInProgressText = taskInProgressText;
-        this.timerText = timerText;
+        this.taskInProgressText = DEFAULT_TASK_IN_PROGRESS;
+        this.timerText = DEFAULT_TIMER;
```

Comment:
oh, why use a String placeholder instead of the actual default time?

Reply:
I plan to set it to default timer once it is set up by Hardy! For now I'm just tracking to make a note that these elements are placeholders.